controllers/controllers.js

```diff
@@ -1,13 +1,43 @@
-const { allCategories } = require("../models/models");
+const { allCategories, returnReview } = require("../models/models");
 
 exports.getCategories = (req, res, next) => {
   const { slug, description } = req.query;
   allCategories(slug, description)
     .then((categories) => {
-        console.log(categories)
       res.status(200).send({ categories });
     })
     .catch((err) => {
```

Comment:
If you're using next in your controllers and the code gets to the catch block, there has absolutely been an error. You do not need a callback function checking if there is one. Simply passing next to the catch block is sufficient.